index.Rmd

```diff
@@ -145,6 +146,7 @@ anim2<-ggplot() +
 
 anim2
 
+
```

Comment:
I don't think your animated graphs showed successfully on the website. Looking forward to seeing them!